apps/nextjs/src/components/screens/WishList/WishListScreen.tsx

```diff
@@ -61,6 +62,18 @@ export const WishListScreen = (props: WishListScreenProps) => {
               buttonName="Add a wish"
               onSubmit={onSubmit}
             />
+            {wishes && wishes.length !== 0 ? (
+              <WishExportModal
```

Comment:
It would be good if you could add some spacing between the download button and the current add a wish button.

In the future we can explore other ways of placing the download button together with to be created functionality (share wish list - link, filtering of items ...)